<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Stadium</class>
 <widget class="QWidget" name="Stadium">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/background/elements/stadium.jpg);</string>
  </property>
  <widget class="QLabel" name="pokemonImg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>281</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/pokemon/elements/Pikachu.png);
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="enemyImg">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>200</y>
     <width>281</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/pokemon/elements/Pikachu.png);
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="textOurPokemon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border: none;
font: 36pt &quot;Wawati SC&quot;;</string>
   </property>
   <property name="text">
    <string>我方宝可梦</string>
   </property>
  </widget>
  <widget class="QLabel" name="textEnemyPokemon">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>0</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border: none;
font: 36pt &quot;Wawati SC&quot;;</string>
   </property>
   <property name="text">
    <string>敌方宝可梦</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="pokemonHp">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="enemyHp">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>160</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="invertedAppearance">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="pokemonName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border: none;
font: 700 48pt &quot;Hannotate SC&quot;;</string>
   </property>
   <property name="text">
    <string>精灵名</string>
   </property>
  </widget>
  <widget class="QLabel" name="enemyName">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>70</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border: none;
font: 700 48pt &quot;Hannotate SC&quot;;</string>
   </property>
   <property name="text">
    <string>精灵名</string>
   </property>
  </widget>
  <widget class="QLabel" name="textVS">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border: none;
font: 700 64pt &quot;HanziPen SC&quot;;</string>
   </property>
   <property name="text">
    <string>VS.</string>
   </property>
  </widget>
  <widget class="QLabel" name="pokemonLv">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>490</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border: none;
font: 36pt &quot;Wawati SC&quot;;</string>
   </property>
   <property name="text">
    <string>Lv.</string>
   </property>
  </widget>
  <widget class="QLabel" name="enemyLv">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>490</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border: none;
font: 36pt &quot;Wawati SC&quot;;</string>
   </property>
   <property name="text">
    <string>Lv.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pokemonAbility">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>600</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border: none;
font: 36pt &quot;Wawati SC&quot;;</string>
   </property>
   <property name="text">
    <string>技能：</string>
   </property>
  </widget>
  <widget class="QLabel" name="enemyAbility">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>600</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border: none;
font: 36pt &quot;Wawati SC&quot;;</string>
   </property>
   <property name="text">
    <string>技能：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="textWe">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>490</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border: none;
font: 36pt &quot;Wawati SC&quot;;</string>
   </property>
   <property name="text">
    <string>我方</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="textName">
   <property name="geometry">
    <rect>
     <x>507</x>
     <y>540</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border: none;
font: 36pt &quot;Wawati SC&quot;;</string>
   </property>
   <property name="text">
    <string>精灵名</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="textUse">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>580</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border: none;
font: 36pt &quot;Wawati SC&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;使用了&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="textEffect">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>640</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border: none;
font: 36pt &quot;Wawati SC&quot;;</string>
   </property>
   <property name="text">
    <string>效果拨群！</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="turnNum">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>110</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border: none;
font: 36pt &quot;Wawati SC&quot;;</string>
   </property>
   <property name="text">
    <string>第0回合</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pokemonHpLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border: none;
font: 36pt &quot;Wawati SC&quot;;</string>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLabel" name="enemyHpLabel">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>110</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border: none;
font: 36pt &quot;Wawati SC&quot;;</string>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
